qa/integration-tests/src/test/java/io/camunda/it/client/ProcessInstanceAndFlowNodeInstanceSearchTest.java

```diff
@@ -613,16 +616,16 @@ void shouldQueryProcessInstancesByVariableAllVariablesMustMatch() {
         List.of(
             new ProcessInstanceVariableFilterRequest()
                 .name("xyz")
-                .value(new StringPropertyImpl().eq("\"bar\"").build()),
+                .value(new StringFilterProperty().$eq("\"bar\"")),
```

Comment:
❓ Can you explain why we use the REST DTO class here to write the tests and then map them back to the client DTO class? Can't we just use the client DTO class to write the tests? That is what the users will also do in the end.

Reply:
Let's create a follow-up issue/PR for this improvement idea, as [described below](https://github.com/camunda/camunda/pull/29792#pullrequestreview-2731188919).